app/src/main-process/main.ts

```diff
@@ -431,6 +433,50 @@ app.on('ready', () => {
     })
   })
 
+  ipcMain.on('setup-auto-updater', () => {
+    mainWindow?.setupAutoUpdater()
+  })
+
+  ipcMain.handle('check-for-updates', async (_, url) => {
+    return mainWindow?.checkForUpdates(url)
+  })
+
+  ipcMain.on('quit-and-install-updates', () => {
+    mainWindow?.quitAndInstallUpdate()
+  })
+
+  ipcMain.on('dispose-auto-updater', () => {
+    mainWindow?.disposeAutoUpdater()
+  })
+
+  ipcMain.on('subscribe-native-theme-updated', () => {
+    mainWindow?.subscribeNativeThemeUpdated()
+  })
+
+  ipcMain.on('minimize-window', () => {
+    mainWindow?.minimizeWindow()
+  })
+
+  ipcMain.on('maximize-window', () => {
+    mainWindow?.maximizeWindow()
+  })
+
+  ipcMain.on('unmaximize-window', () => {
+    mainWindow?.unmaximizeWindow()
+  })
+
+  ipcMain.on('close-window', () => {
+    mainWindow?.closeWindow()
+  })
```

Comment:
We can tighten all of these (and several of the other as well) up by using the short hand arrow function syntax
```suggestion
  ipcMain.on('minimize-window', () => mainWindow?.minimizeWindow())
  ipcMain.on('maximize-window', () => mainWindow?.maximizeWindow())
  ipcMain.on('unmaximize-window', () => mainWindow?.unmaximizeWindow())
  ipcMain.on('close-window', () => mainWindow?.closeWindow())
```